cli/azd/pkg/tools/powershell/powershell.go

```diff
@@ -3,31 +3,32 @@ package powershell
 import (
 	"context"
 
+	"github.com/azure/azure-dev/cli/azd/pkg/environment"
```

Comment:
same comment than `bash` package.

`environment.Env` -> describes what an azd environment is like.
`bash.Env` -> what a bash env is like.
`powershell.Env` -> what a power shell env should look like.